modules/instrument_list/templates/menu_instrument_list.tpl

```diff
@@ -57,13 +55,11 @@
   <tbody>
   <tr>
     <td>
-      {$display.DoB}
+      {$dob_age}
```

Comment:
change it like this     {if $dob_age!=""}
      <td>
        {$dob_age}
      </td>
    {/if}

Reply:
same as edc